src/eva/tests/config/testGsiObsSpaceAmsuaMetop-A.yaml

```diff
@@ -0,0 +1,21 @@
+diagnostics:
+
+  # Data read
+  # ---------
+- data:
+    type: GsiObsSpace
+    datasets:
+      - name: experiment
+        satellite: metop-a
+        sensor: amsua
+        filenames:
+          - /scratch2/NCEPDEV/stmp1/Kevin.Dougherty/ncDiags/gdas.20200922/00/atmos/diag_amsua_metop-a_ges.2020092200.nc4
```

Comment:
These would need to be relative paths to the files downloaded by Git-LFS rather than on Hera.